en/contributing/build.md

```diff
@@ -82,37 +87,43 @@ To build the *DroneCore* C++ Library on Windows:
 
 
 ## Install DroneCore {#install-artifacts}
 
-In order to *use* DroneCore your C++ applications need to be able locate the DroneCore libraries and header files (see [Building C++ Apps](../guide/toolchain.md)). You can either install these to standard system locations or locally within the DroneCore directory.
-
-> **Tip** The example code assumes that DroneCore is installed locally. The example **CMakeLists.txt** file can easily be modified to find the system-wide installation.
+*Installing* builds DroneCore **and** copies the libraries and header files into a "public" location so that they can be referenced by C++ applications (see [Building C++ Apps](../guide/toolchain.md)). DroneCore supports installation system-wide (recommended) or locally within the DroneCore tree.
 
+> **Warning** System-wide installation is not yet supported on Windows (see [#155](https://github.com/dronecore/DroneCore/issues/155)) so you will need to [install DroneCore locally](#dronecore_local_install).
+>
+> Windows gurus, we'd [love your help](../README.md#getting-help) to implement this).
 
-### System-wide Install
+### System-wide Install {#dronecore_system_wide_install}
 
-Installing DroneCore system-wide means that the files you need to build your apps are always in the same place, and your build definition files can be simpler because they do not need to consider relative locations.
+System-wide installation copies DroneCore to the standard system-wide locations for your platform (On Ubuntu Linux this is **/usr/local/**).
 
-> **Warning** System-wide install is not (yet) supported on Windows. If you're a Windows guru, we'd [love your help](../README.md#getting-help) to set this up.
+> **Warning** System-wide installation overwrites any previously installed version of DroneCore. 
 
-To install the files system-wide:
+To install DroneCore system-wide:
 
 ```sh
-make clean  #REQUIRED!
-sudo INSTALL_PREFIX=/usr/local make default install
+sudo make clean  #REQUIRED!
+sudo default install
 ```
 
-> **Note** System-wide install overwrites standard install locations. If you already have DroneCore installed through some other mechanism it will be replaced!
+> **Tip** The `sudo` command is required (above) in order to copy or remove files in system directories.
+
+### Local Install {#dronecore_local_install}
 
-### Local Install
+Local installation copies DroneCore headers/library to a user-specified location in the DroneCore directory.
 
-On Linux/macOS the DroneCore headers and static library can be installed locally into the folder **DroneCore/install/** using:
+On Linux/macOS use the `INSTALL_PREFIX` variable to specify a path relative to the **DroneCore/build/** folder
+(or an absolute path).
+For example, to install into the **DroneCore/install/** folder you would call:
 
 ```sh
-make default install
+make clean  #REQUIRED!
+INSTALL_PREFIX=../../install make default install
 ```
 
 On Windows specify `--target install` when building, as shown:
```

Comment:
Does this windows command work? Is it still "local" and is the `install` in `--target install` an instruction or a folder name?

Reply:
I have no idea at this point sorry. I don't understand windows enough.

Reply:
No worries. This is broken until #55 is fixed, and then we can look at it.

Reply:
Just FYI, it is a target name - so always installs the "right" place. Since we only support local install that is /DroneCore/install/

Reply:
Hm, ok. So cmake does not install system wide on Windows then.

Reply:
No, at least not now. I would really like it to, because at the moment our instructions are all "forky". Would be great to have install system wide as default and a separate document for local install for "experts" - but we can't do that while windows requires local.